### ISSUE TITLE
Please update Winget

### ISSUE THREAD

**Issue by TheDevelolper** [anchore/grype#2270]
**What would you like to be added**:
Please update winget

![image](https://github.com/user-attachments/assets/b8ccfa9b-4795-4263-a96d-532cfe1c566e)

**Why is this needed**:
For obvious reasons 😂

**Additional context**:

![image](https://github.com/user-attachments/assets/9778392c-27cd-4d98-8881-209ae4682977)

**Comment by popey** [comment]
Hi @TheDevelolper - thanks for the issue! Also, thanks for the opportunity for me to learn how WinGet packages are updated! I installed [Komac](https://github.com/russellbanks/Komac/releases) (on my arm64 mac!), and did this:

Check the versions in winget:

```
$ komac list-versions Anchore.Grype
0.69.0
0.74.2
0.79.1
```

Forked [microsoft/winget-pkgs](https://github.com/microsoft/winget-pkgs) to [popey/winget-pkgs](https://github.com/popey/winget-pkgs)

Attempt to update Grype:

```
$ komac update Anchore.Grype  --version 0.84.0 --urls https://github.com/anchore/grype/releases/download/v0.84.0/grype_0.84.0_windows_amd64.zip
Latest version of Anchore.Grype: 0.79.1
# Created with komac v2.6.0
# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json

PackageIdentifier: Anchore.Grype
PackageVersion: 0.84.0
InstallerType: zip
NestedInstallerType: portable
NestedInstallerFiles:
- RelativeFilePath: grype.exe
UpgradeBehavior: install
ReleaseDate: 2024-11-05
Installers:
- Architecture: x64
  InstallerUrl: https://github.com/anchore/grype/releases/download/v0.84.0/grype_0.84.0_windows_amd64.zip
  InstallerSha256: B8670FAEDCB3C993F09C31DB184EBDA5EADD4120891419C420DBE7527507307C
ManifestType: installer
ManifestVersion: 1.6.0

# Created with komac v2.6.0
# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json

PackageIdentifier: Anchore.Grype
PackageVersion: 0.84.0
PackageLocale: en-US
Publisher: Anchore Inc
PublisherUrl: https://anchore.com/
PublisherSupportUrl: https://github.com/anchore/grype/issues
PackageName: Grype
PackageUrl: https://github.com/anchore/grype
License: Apache-2.0
LicenseUrl: https://github.com/anchore/grype/blob/HEAD/LICENSE
ShortDescription: A vulnerability scanner for container images and filesystems
Description: |-
  A vulnerability scanner for container images and filesystems. Easily install the binary to try it out.
  Works with Syft, the powerful SBOM (software bill of materials) tool for container images and filesystems.
Moniker: grype
Tags:
- container-image
- containers
- cyclonedx
- docker
- go
- golang
- hacktoberfest
- oci
- openvex
- security
- static-analysis
- tool
- vex
- vulnerabilities
- vulnerability
ReleaseNotes: |-
  Added Features
  - Add support for scanning single purl from the CLI [#2225 #2223 @wagoodman]
  Bug Fixes
  - Docker reports 0 vulnerabilities.  Same file reports many vulnerabilites when ran directly on linux server [#2235]
  - Flaky checks on STDIN for purl provider [#2192 #2223 @wagoodman]
  - Missing alpine patch version yields inaccurate results [#2222 #2226 @wagoodman]
  Additional Changes
  - update Syft to v1.16.0 [#2237 @anchore-actions-token-generator]
  (Full Changelog)
ReleaseNotesUrl: https://github.com/anchore/grype/releases/tag/v0.84.0
ManifestType: defaultLocale
ManifestVersion: 1.6.0

# Created with komac v2.6.0
# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json

PackageIdentifier: Anchore.Grype
PackageVersion: 0.84.0
DefaultLocale: en-US
ManifestType: version
ManifestVersion: 1.6.0

> What would you like to do with Anchore.Grype 0.84.0? Submit
Successfully created a pull request to microsoft/winget-pkgs
https://github.com/microsoft/winget-pkgs/pull/192597
```

Now, we wait...

**Comment by popey** [comment]
Thanks again @TheDevelolper - the 0.84.0 release was successfully published, and today's 0.85.0 is in the queue via [this pr](https://github.com/microsoft/winget-pkgs/pull/192894).

We'll keep on top of this going forward.

**Comment by TheDevelolper** [comment]
Don't thank me, I just reported it. Thank you for actioning it so quickly.
